server/controllers/finance/reports/cashReport/report_separated.handlebars

```diff
@@ -0,0 +1,133 @@
+                <th style="width: 10%"> {{ translate 'TABLE.COLUMNS.SOURCE'  }} </th>
+                <th style="width: 10%"> {{ translate 'TABLE.COLUMNS.BY' }} </th>
+                <th style="width: 10%"> {{ translate 'TABLE.COLUMNS.AMOUNT' }} </th>              
+              </tr>            
+            </thead>
+            <tbody>
+              {{#each this.entries}}
+                <tr>
+                  <td class="text-left">{{ trans_id }}</td>
+                  <td class="text-left">{{ date trans_date }}</td>              
+                  <td class="text-left" style="white-space : nowrap;  text-overflow : ellipsis; overflow : hidden; max-width : 500px;">
+                    {{ description }}
+                  </td>
+                  <td class="text-left" style="white-space : nowrap;  text-overflow : ellipsis; overflow : hidden;">
+                    {{ translate transactionType }}
+                  </td>
+                  <td class="text-left">{{ username }}</td>
+                  <td class="text-right">
+                    {{#if debit}}
+                      {{ currency debit ../metadata.enterprise.currency_id}}
+                    {{/if}}  
+                  </td>
+                </tr> 
+              {{/each}}
+            </tbody>
+            <tfoot>
+              <tr style="background-color:#ddd;">
+                <th class="text-right" colspan="6"> 
+                  {{ translate 'TABLE.COLUMNS.TOTAL' }} : {{ currency this.totalEntry metadata.enterprise.currency_id}}
+                </th>
+              </tr>
+            </tfoot>
+          </table>
+        {{/if}}
+        {{#if (isExpenseViewable this.type_id)}}
+          <h6 class="text-center text-uppercase">
+            {{ translate 'REPORT.CASH_EXPENSE' }}
+          </h6>
+          <table class="table table-condensed table-report">
+            <thead>
+              <tr style="background-color:#ddd;">
+                <th style="width: 10%"> {{ translate 'TABLE.COLUMNS.TRANSACTION_ID' }} </th>
+                <th style="width: 10%"> {{ translate 'TABLE.COLUMNS.DATE'  }} </th>              
+                <th style="width: 50%"> {{ translate 'TABLE.COLUMNS.DESCRIPTION'}} </th>
+                <th style="width: 10%"> {{ translate 'TABLE.COLUMNS.SOURCE'  }} </th>
+                <th style="width: 10%"> {{ translate 'TABLE.COLUMNS.BY' }} </th>                            
+                <th style="width: 10%"> {{ translate 'TABLE.COLUMNS.AMOUNT' }} </th>
+              </tr>            
+            </thead>
+            <tbody>
+              {{#each this.expenses}}
+                <tr>
+                  <td class="text-left">{{ trans_id }}</td>
+                  <td class="text-left">{{ date trans_date }}</td>              
+                  <td class="text-left" style="white-space : nowrap;  text-overflow : ellipsis; overflow : hidden; max-width : 500px;">
```

Comment:
At some point we should make this into a class - something like `text-overflow-hidden`.  You use it a lot.